perf.groovy

```diff
@@ -100,6 +103,19 @@ def static getOSGroup(def os) {
 					}
 				}
 			
+			if (isPR)
+			{
+				parameters
+				{
+					stringParam('CommitName', '%ghprbPullTitle%', 'The name of the commit for when we insert the performance data into the database.')
```

Comment:
For Windows it should be %, but in the Ubuntu case I do need to change it to the escaped dollar sign.